ui/k-info-panel/list_maintenance.kytos

```diff
@@ -0,0 +1,363 @@
+<template>
+    <div v-if="render_value > 0">
+        <div class="maintenance_container">
+            <div class="maintenance-table no-compact">
+                <div class="maintenance-table-divisor">
+                    <table id="maintenance-table-list-windows">
+                        <thead>
+                            <tr>
+                                <template v-for="(header, index) in table_headers">
+                                    <th scope="col" class="header" @click="sort(index)">
+                                        {{ header }}
+                                        <span class="order-dir" v-if="current_sort === index">
+                                            {{ spanDir(index) }}
+                                        </span>
+                                    </th>
+                                </template>
+                            </tr>
+                            <tr class="header-filter">
+                                <template v-for="header in table_headers">
+                                    <th scope="col">
+                                        <input :id="header + '_search'" v-model="search_terms[header]"></input>
+                                    </th>
+                                </template>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <tr v-for="row in rowsOfPage">
+                                <template v-for="column in row">
+                                    <td>{{column}}</td>
+                                </template>
+                            </tr>
+                        </tbody>
+                    </table>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div v-else>
+        <div class="maintenance_container">
+            <p class='empty-window-list' v-if='loading'>Loading maintenace windows. Please wait...</p>
+            <p class='empty-window-list' v-else>No maintenance windows available.<p>
+        </div>
+    </div>
+</template>
+
+<script>
+ module.exports = {
+   data () {
+     return {
+       table_headers: ['Id', 'Start', 'End', 'Description', 'Links', 'Switches', 'Interfaces', 'Status'],
```

Comment:
@Aduan002, the multi column split for `items` turned out really great. Regarding CSS, and the columns displayed it and assuming an average monitor resolution like 1080x1920 the horizontal scroll can show up relatively easily, which doesn't impact much, but:

![2022-02-23-150515_1919x311_scrot](https://user-images.githubusercontent.com/1010796/155381271-706dc8c0-c00f-452c-aff6-c3123e39f12b.png)

1 - I wonder if the table columns could be presented in this order: `['Id', 'Start', 'End', 'Description', 'Status', 'Links', 'Switches', 'Interfaces']`, trying to present `Status` as early as we can before the `items` since this information is very important for operators without having to scrolling, wdyt?
2 - I wonder if you could try to slightly reduce the font size to see if might help out a bit with this. 

and finally, your PR will land and be scheduled for the next release, so, could you also add a line in the [`CHANGELOG.rst`](https://github.com/kytos-ng/maintenance/blob/master/CHANGELOG.rst) describing this change under the `unreleased - under development` section? something along these lines:

```
[UNRELEASED] - Under development
********************************

Added
=====
- Added UI `k-toolbar` component to list maintenance windows
- Added UI `k-info-panel` component to list maintenance windows in a sortable and filterable table
```

Reply:
Ok. I added these new changes.

Reply:
Awesome. Looks good to me, thanks for pushing this commit, I'll mark this thread as resolved. 

![2022-02-23-182423_1919x297_scrot](https://user-images.githubusercontent.com/1010796/155411788-1d67f2b1-d94c-45dc-92ea-2a3514a9f0e3.png)